keyboards/projectcain/vault45/keymaps/default/config.h

```diff
@@ -0,0 +1,6 @@
+#define ENCODERS_PAD_A { B3, D1 }
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Additionally: 
```suggestion
#pragma once

#define ENCODERS_PAD_A { B3, D1 }
```